### PR TITLE
Updated theme-utils.mjs help text

### PR DESCRIPTION
**Pull request by markbiek**
#### Changes proposed in this Pull Request:

Running `node theme-utils.mjs` (no arguments) or `node theme-utils.mjs
help` will now show a list of available commands.

Running `node theme-utils.mjs help <command>` will show the specific
help text for that command.

image

### CODE REVIEW

File: theme-utils.mjs [hunk 1]
```diff
@@ -34,13 +34,166 @@ const coreThemes = ['twentyten', 'twentyeleven', 'twentytwelve', 'twentythirteen
...
+				additionalArgs = '<theme-slug> <add-changes, true/false>'
+				helpText = 'Use the commit log to build a list of recent changes and add them as a new changelog entry. If add-changes is true, the updated readme.txt will be staged.';
+				break;
+			}
+		case "rebuild-theme-changelog":
+			{
+				additionalArgs = '<theme-slug> <since>';
+				helpText = 'Rebuild the entire change long from the given starting hash.';
+				break;
+			}
```
Comment (madhusudhand) [inline theme-utils.mjs]: This is super helpful.

I have the following thoughts in the view of maintaining all commands from a single place.

Turn the information into an object. (also considering the switch case in line 17-38)
```js
const commands = {
  "push-button-deploy": { helpText: '',  additionalArgs: '', run: () =>  pushButtonDeploy() },
  "deploy-sync-core-theme": { helpText: '',  additionalArgs: '', run: () =>  deploySyncCoreTheme(args?.[1], args?.[2]) },
  ...
}
```

now the switch cases could be replaced by
```
// line 17-38
commands[command].run();

// line 75-190
const {helpText, additionalArgs} = commands[command];
```

Also `showHelp` method can be simplified to

```js
return `Available commands` + Object.keys(commands).join('\n')
```

Comment (markbiek) [reply theme-utils.mjs]: I like that! Much tidier looking!

Thanks for the suggestion, I should be able to get that taken care of today.

File: theme-utils.mjs [hunk 2]
```diff
@@ -8,39 +8,141 @@ const sandboxPublicThemesFolder = '/home/wpdev/public_html/wp-content/themes/pub
...
 const isWin = process.platform === 'win32';
-const premiumThemes = [ 'videomaker', 'videomaker-white' ];
+const premiumThemes = ['videomaker', 'videomaker-white'];
 const coreThemes = ['twentyten', 'twentyeleven', 'twentytwelve', 'twentythirteen', 'twentyfourteen', 'twentyfifteen', 'twentysixteen', 'twentyseventeen', 'twentynineteen', 'twentytwenty', 'twentytwentyone', 'twentytwentytwo'];
 
+const commands = {
+	"push-button-deploy": {
+		helpText: `
+* Gets the last deployed hash from the sandbox
+* Version bump all themes have have changes since the last deployment
```
Comment (sixhours) [inline theme-utils.mjs]: "Version bump all themes *that* have changes..." I think?

Comment (markbiek) [reply theme-utils.mjs]: Good catch! That was in the original comment on the handler function and I didn't see it 😝

**Review by valterlorran — APPROVED** [review]
👍 Looks good.

**Review by madhusudhand — APPROVED** [review]
LGTM!

**Pull request merged**